pkgs/applications/networking/p2p/zeronet/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, buildPythonApplication
+, python, msgpack, gevent }:
+
+buildPythonApplication rec {
+  pname = "zeronet";
+  version = "0.6.2";
+
+  src = fetchFromGitHub {
+    owner = "HelloZeroNet";
+    repo = "ZeroNet";
+    rev = "v${version}";
+    sha256 = "0v19jjirkyv8hj2yfdj0c40zwynn51h2bj4issn5blr95vhfm8s7";
+  };
+
+  propagatedBuildInputs = [ msgpack gevent ];
+
+  format = "other";
+ 
+  buildPhase = "python -O -m compileall .";
+
+  installPhase = ''
+    mkdir -p $out/share
+    cp -r plugins src tools *.py $out/share/
+  '';
+
+  # Wrap the main executable and set the log and data dir to something out of
+  # the store
+  fixupPhase = ''
```

Comment:
Its intentional to replace any existing `fixupPhase`?

Reply:
But there shouldn't be any, as I am using `format="other"`. Or should I move it to postFixup anyway?

Reply:
Oh well, moving it won't do any harm